Gem/Code/Source/Automation/ScriptReporter.h

```diff
@@ -88,6 +88,10 @@ namespace AtomSampleViewer
         // For exporting test results
         void ExportTestResults();
 
+        // Path to the exported test results file.
+        AZStd::string m_exportedTestResultsPath = "Click the 'Export Test Results' button.";
+        AZStd::string GetExportedTestResultsPath() const;
```

Comment:
Can we rename this to GenerateExportedTestResultsPath() since we are essentially creating a unique filename dynamically?